update-version.js

```diff
@@ -3,6 +3,11 @@ const fs = require("fs");
 const path = "./package.json";
 
 const package = fs.readFileSync(path, { encoding: "utf8", flag: "r" });
-const updated = package.replaceAll("1.59.0", "1.63.0");
+const json = JSON.parse(package);
+json.contributes.configuration.properties[
+  "metals.suggestLatestUpgrade"
+].default = true;
+json.devDependencies["@types/vscode"] = "1.63.0";
```

Comment:
You have to also alter this `json.engines.vscode = "1.63.0";`

Reply:
thx. fixed